app/src/main/kotlin/com/hippo/ehviewer/ui/reader/ReaderScreen.kt

```diff
@@ -62,11 +65,12 @@ import moe.tarsin.kt.unreachable
 @Composable
 fun ReaderScreen(info: BaseGalleryInfo, page: Int = -1) {
     LockDrawer(true)
-    val pageLoader = remember {
+    val cropBorder by Settings.cropBorder.collectAsState()
+    val pageLoader = remember(cropBorder) {
```

Comment:
Recreating pageloader is expensive, just restart would be enough.

Reply:
Restart pageLoader will not reset pages' state, let's keep it as is until we drop ReaderActivity and refactor Pageloader to compose style